main.tf

```diff
@@ -382,6 +382,7 @@ resource "aws_codebuild_project" "default" {
       content {
         name  = "GITHUB_TOKEN"
         value = var.github_token
+        type  = "SECRETS_MANAGER"
```

Comment:
This would be better as an input variable.

Reply:
Any rational for allowing developer to not store this token securely?

Reply:
`PARAMETER_STORE` can also be used to store secrets securely.